### ISSUE TITLE
Move class Cell into its own file: website/nb_cell.ts

### ISSUE THREAD

**Issue by ry** [propelml/propel#361]
currently in website.nb.ts

**Comment by curt-mitch** [comment]
Can I take this one? I'm assuming the new file should now become website/nb_cell.tsx

**Comment by ry** [comment]
Yes, thanks. but #441 should land first and I think has nb changes

**Comment by curt-mitch** [comment]
I'll wait for a merge or update on https://github.com/propelml/propel/pull/480 before opening a PR for this.

**Comment by ry** [comment]
@curt-mitch Yeah sorry about the delay - I spent sometime trying to get that test working to no avail - let's just merge your patch now. Hopefully we can fix that race condition later.
